src/context.ts

```diff
@@ -634,6 +634,45 @@ export default class Context {
 		return this._customFieldKeys();
 	}
 
+	private _customFieldValue(experimentName: string, key: string): JSONValue {
+		const experiment = this._index[experimentName];
+
+		if (experiment != null) {
+			const field = experiment.data.customFieldValues?.find((x) => x.name === key);
+			if (field != null) {
+				switch (field.type) {
+					case "text":
+					case "string":
+						return field.value;
+					case "number":
+						return Number(field.value);
+					case "json":
+						try {
+							return JSON.parse(field.value);
```

Comment:
I think JSON.parse will fail for empty strings and null. Maybe we shouldn't error in that case?

Reply:
Null seems to be okay, but strings aren't. I'll change it :)
<img width="538" alt="image" src="https://github.com/absmartly/javascript-sdk/assets/30095183/ddb25c37-0df4-4671-a5f1-a0818476dd76">